### ISSUE TITLE
Improve notifications for image validation problems

### ISSUE THREAD

**Issue by indigoxela** [backdrop-contrib/ckeditor5#97]
It seems that currently we have nothing at all.

Image uploads via paste might need the ability to give a hint, why an image failed to upload (too big, unsupported type...).

Currently, whatever I drop into the editor, **no** message appears. The only exception is when images get resized. An (improvised) alert shows the info (not merged yet, see issue #95).

CKE4 does provide an API for notifications, and they're styled:

![cke4-notification](https://github.com/backdrop-contrib/ckeditor5/assets/842066/71786698-b4b0-469a-b7b5-acf489a7b863)

CKE5 doesn't provide comparable functionality (it seems) https://ckeditor.com/docs/ckeditor5/latest/api/module_ui_notification_notification-Notification.html

**Comment by quicksketch** [comment]
Yeah, it seems right now CKEditor 5's notification system currently only provides other plugins the ability to respond to notifications, but there is no notification UI yet (there's an issue for that at https://github.com/ckeditor/ckeditor5/issues/8934). Right now it just falls back to `window.alert()`, resulting in the same behavior as we've set up in #95.
